src/storage/async_page_fetcher.cpp

```diff
@@ -114,9 +114,18 @@ namespace cublog
 
   void log_boot_info_fetch_task::execute (context_type &context)
   {
-    std::string message = log_pack_log_boot_info (&context);
+    struct log_lsa append_lsa, prev_lsa;
```

Comment:
Having to mention struct before a type was a C thing. It is no longer required.
```suggestion
    log_lsa append_lsa, prev_lsa;
```

Reply:
Done